_maps/map_files/cerestation/cerestation.dmm

```diff
@@ -95725,13 +95725,14 @@
 /turf/simulated/floor/carpet/royalblack,
 /area/station/command/office/captain)
 "xNF" = (
-/obj/machinery/door/window{
-	dir = 8;
-	name = "Robotics Surgery"
-	},
 /obj/machinery/atmospherics/unary/vent_pump/on{
 	dir = 1
 	},
```

Comment:
It also looks like there was never an access helper on this windoor in the first place. Could you add that as part of this PR?

Reply:
Fixed as well 
![Screenshot 2024-01-06 163929](https://github.com/ParadiseSS13/Paradise/assets/98861947/3ffc06fc-26d1-4df7-b6d4-c139d5a39383)

Reply:
with both robotics airlock helpers